### PR TITLE
Add "Served from the Cloud" page

### PR DESCRIPTION
**Pull request by cbf66**

### CODE REVIEW

File: source/help/cloud.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Served from the cloud
+===============
+
+For approximately the last 20 years, arXiv's services have been hosted on a set of services hosted at Cornell's central IT facility in Ithaca New York. In recent years, we've approached the limits of what a University IT facility can offer, and have thus begun to shift our services to a public cloud vendor.
```
[the remark this answers is not in the document]
Comment (cbf66) [reply source/help/cloud.md:4]: @bdc34 -- that's not grammatical ?

[the remark this answers is not in the document]
Comment (cbf66) [reply source/help/cloud.md:4]: You're saying to cut out ", we've approached the limits of what a University IT facility can offer," ?

File: source/help/cloud.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+We are doing this process little by little, starting with the browsing activities &ndash; serving the home page, listings, abstracts, and most importantly, article downloads (PDF and other formats). The portions of the arXiv site that enable users to change things (account creation and mainteance, submiting and updating articles) will transition later.
```
Comment (bdc34) [inline source/help/cloud.md:6]: Simplify:

The transition will happen later for account creation and maintenance, submitting and updating articles, and other pages.

Also check spelling of maintenance and submitting.

File: source/help/cloud.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bmaltzan) [reply source/help/cloud.md:4]: > arXiv's services have been hosted on a set of services hosted at Cornell's central IT facility in Ithaca New York
arXiv's services have been hosted at Cornell's IT facility in Ithaca New York

[the remark this answers is not in the document]
Comment (bdc34) [reply source/help/cloud.md:4]: Yes, cut out that part.  "Recently we have begun to shift our services to a public cloud vendor."

**Review by bdc34 — APPROVED** [review]
I'm for this with or without the suggested changes.

File: source/help/cloud.md (hunk 2 above)
Comment (bmaltzan) [reply source/help/cloud.md:6]: Maybe think ahead to search; readers may have a different understanding of what includes browsing; and put the most important thing first:

> starting with the browsing activities &ndash; serving the home page, listings, abstracts, and most importantly, article downloads (PDF and other formats).

starting with PDFs, abstracts, listings, and search.

**Comment by cbf66** [comment]
Add "Served from the Cloud" page

**Comment by cbf66** [comment]
Add "Served from the Cloud" page

**Review by alisonhofer — COMMENTED** [review]
OK!

**Pull request merged**